### PR TITLE
Add crop image demo for Upload

### PR DESCRIPTION
**Pull request by nanxiaobei**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [x] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Perfermance optimization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |           |
| 🇨🇳 Chinese |           |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

-----
[View rendered components/upload/demo/crop-image.md](https://github.com/nanxiaobei/ant-design/blob/master/components/upload/demo/crop-image.md)

### CODE REVIEW

**Comment by xrkffgg** [comment]
提交到 feature 分支

**Comment by afc163** [comment]
不用提交到 feature，只是一个演示。

File: package.json [hunk 1]
```diff
@@ -106,6 +106,7 @@
     "@ant-design/css-animation": "^1.7.2",
     "@ant-design/icons": "^4.1.0",
     "@ant-design/react-slick": "~0.26.1",
+    "antd-img-crop": "^3.1.1",
```
Comment (afc163) [inline package.json:109]: `devDependencies`

Comment (nanxiaobei) [reply package.json]: 已更新

**Comment by afc163** [comment]
`npm test -- -u` to update test snapshot.

**Comment by afc163** [comment]
同步一下上游主干。

**Comment by xrkffgg** [comment]
rebase 后
git push -f

**Pull request merged**